errors/missing-env-file.md

```diff
@@ -2,7 +2,7 @@
 
 #### Why This Error Occurred
 
-You ran `vercel dev` inside a project that contains a `vercel.json` file with `env` or `build.env` properties that use [Vercel Secrets](https://vercel.com/docs/v2/build-step#environment-variables).
+You ran `vercel dev` inside a project that contains a `vercel.json` file with `env` or `build.env` properties that use [Vercel Secrets](https://vercel.com/docs/concepts/projects/environment-variables).
```

Comment:
it looks like someone created a tiny section at https://vercel.com/docs/concepts/deployments/build-step#environment-variables which is simply a link to https://vercel.com/docs/concepts/projects/environment-variables

Seems like it could have been a redirect instead.

Reply:
I would change it to "Environment Variables", but happy to defer to whomever can make these decisions.

Reply:
Looks like this error really is for secrets, not all env vars. You can see it checks for “@“ here https://github.com/vercel/vercel/blob/da893e7c573ec35308e542287108a6a80f9896a6/packages/cli/src/util/dev/server.ts#L1137

That makes me think we need a new section in the docs for secrets, or a proper deprecation.